spec/requests/image_uploads_spec.rb

```diff
@@ -99,15 +98,20 @@
         expect(cache.read(cache_key).to_i).to eq(1)
       end
 
-      it "raises error with too many uploads" do
+      it "responds with HTTP 429 with too many uploads" do
         upload = proc do
           Rack::Test::UploadedFile.new(
-            Rails.root.join("spec/support/fixtures/images/images1.jpeg"), "image/jpeg"
+            Rails.root.join("spec/support/fixtures/images/image1.jpeg"),
+            "image/jpeg",
           )
         end
-        expect do
-          11.times { post "/image_uploads", headers: headers, params: { image: [upload.call] } }
-        end.to raise_error(RuntimeError)
```

Comment:
we were raising `RateLimitChecker::UploadRateLimitReached` and sending back a JSON response in `ImageUploadsController` but this test was never changed. The reason why we never noticed it is that the test never failed as it was looking for `images1.jpeg` (instead of `image1.jpeg`) which is a file that does not exist on disk, which eventually raised `RuntimeError` which was compatible with the expectation set. 

👀